scripts/generate-style-json-schema.js

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable no-restricted-syntax */
+
+const fs = require('fs');
```

Comment:
is this at all connected to the SUITE stuff? Would it be better to wait until that work is finished before introducing any of the validation here as it is out of the scope of withThemeStyles?

Reply:
Ooops. Im not sure how that snuck in. Removed.